### ISSUE TITLE
Hi!

### ISSUE THREAD

**Issue by vweevers** [kapetan/indexeddown#1]
Came across this project by chance, nice work. Just wanted to let you know that we (@ralphtheninja and I) have begun to update [level.js](https://github.com/Level/level.js) and it seems `level.js` has the same goals as `indexeddown`.

Only difference (at first glance) is that `level.js` defaults to snapshot guarantees (for full `abstract-leveldown` compliance). What else? Maybe we can join forces!

**Comment by kapetan** [comment]
Hi. The main reasons I wrote this module was because `level.js` was using an old version of `abstract-leveldown` and the iterator always retrieved all stored values even with `limit` set, but I can see that both issues have been fixed by your work.

This module doesn't do any checks to see which version of indexeddb is available, it assumes the most recent version.

With the fixes you made, it might make this library obsolete.
